src/app/summary/summary.service.ts

```diff
@@ -0,0 +1,78 @@
+import {Injectable, OnDestroy} from '@angular/core';
+import {map} from "rxjs/operators";
+import {Subject} from "rxjs";
+import {ProjectsService} from "../projects/services/projects.service";
+
+export interface ProjectCount {
+  group: string;
+  count: number;
+  cellCount: number;
+}
+
+@Injectable({
+              providedIn: 'root'
+            })
+export class SummaryService implements OnDestroy {
+
+  private projectsByOrgan = new Subject<ProjectCount[]>();
+  projectsByOrgan$ = this.projectsByOrgan.asObservable();
+
+  private projectsByTech = new Subject<ProjectCount[]>();
+  projectsByTech$ = this.projectsByTech.asObservable();
+
+  private cellCount = new Subject<number>();
+  cellCount$ = this.cellCount.asObservable();
+
+  constructor(private projectService: ProjectsService) {
+    this.groupProjectsByKey('organs', this.projectsByOrgan);
+    this.groupProjectsByKey('technologies', this.projectsByTech);
+    this.projectService.getAllProjects()
+        .pipe(map(projects => projects.map(x => x.cellCount)))
+        .subscribe(projects => this.cellCount.next(projects.reduce((acc, val) => acc + val, 0)));
+  }
+
+  groupProjectsByKey(key: string, subject: Subject<any>) {
+    this.projectService.getAllProjects()
+        .pipe(
+          map(projects => {
+            let groupedProjects = this.groupListByKey(projects, key);
+            let data: ProjectCount[] = [];
+            this.sortByValue(groupedProjects)
+                ?.forEach(([key, value]) => {
+                  data.push({
+                              count: value.count,
+                              cellCount: value.cellCount,
+                              group: key
+                            });
+                });
+            return data;
+          })
+        )
+        .subscribe(x => subject.next(x),
+                   x => subject.error(x));
+  }
+
+  ngOnDestroy() {
+    this.projectsByOrgan.complete();
```

Comment:
ngOnDestroy is fine if this was not a singleton, but since it is, this will never be called and is uneccesary